### PR TITLE
Unremoves cloning

### PR DESCRIPTION
**Pull request by 4dplanner**
## About The Pull Request
#41852 sneakily removed cloning

Might need a testmerge, I don't understand why coiax WOULD delete those lines in the first place, and that's dangerous.

@coiax 

## Changelog
:cl:
fix: fixes cloning
/:cl:

### CODE REVIEW

**Comment by Cyprex** [comment]
This was the conclusion of >1h of testing on the server by him. Thanks a lot!

**Comment by 4dplanner** [comment]
Please don't tell them how much testing it took, it makes me sound slow ;)

For future coders - show server revision is broken, don't trust it. All my localhost testing/code browsing was on the wrong commit, so nothing made sense.

**Comment by Whoneedspacee** [comment]
@4dplanner I’ve spent more time fixing simpler bugs. I think we all think we take too long but the truth is it’s just not obvious.

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -594,3 +594,7 @@
 	if(old_record)
 		records -= old_record
 		scantemp = "Record updated."
+	else
+		scantemp = "Subject successfully scanned."
+	records += R
+	playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
```
Comment (ShizCalev) [inline code/game/machinery/computer/cloning.dm]: remove the last 0

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by chesse20** [comment]
lol this pr is so close to being 420 69

**Comment by Cyberboss** [comment]
@4dplanner `Show Server Revision` on production using the TGS system which is way more functional than the git system

**Pull request merged**